api/server/router/network/network_routes.go

```diff
@@ -258,7 +258,7 @@ func (n *networkRouter) postNetworkConnect(ctx context.Context, w http.ResponseW
 	// The reason is that, In case of attachable network in swarm scope, the actual local network
 	// may not be available at the time. At the same time, inside daemon `ConnectContainerToNetwork`
 	// does the ambiguity check anyway. Therefore, passing the name to daemon would be enough.
-	return n.backend.ConnectContainerToNetwork(connect.Container, vars["id"], connect.EndpointConfig)
+	return n.backend.ConnectContainerToNetwork(connect.Container, vars["id"], connect.EndpointConfig, true)
```

Comment:
I'm not sure whether this should be set to `false` for API < 1.44 :thinking:

Reply:
If the current order is randomised by go (thus "undefined"), then applying things in the correct order would make it equally "undefined" (just 100% more predictable), so I think it's fine to make it independent of the API version.

The Priority _FIELD_ should not be looked at for older API versions (when creating).

Reply:
> The Priority FIELD should not be looked at for older API versions (when creating).

Mh, I'm assuming `Priority` is 0 for older API versions but that might be only for Go client :thinking: Maybe I should force it to 0 for older versions.